samples/excel/38-pivottable/pivottable-refresh.yaml

```diff
@@ -0,0 +1,139 @@
+order: 8
+id: excel-pivottable-refresh
+name: Refresh
+description: Refreshes a PivotTable based on table row additions.
+host: EXCEL
+api_set:
+    ExcelApi: '1.8'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-table-row").click(() => tryCatch(addTableRow));
+        $("#refresh-pivottable").click(() => tryCatch(refreshPivotTable));
+
+        async function addTableRow() {
+          await Excel.run(async (context) => {
+            // Add a row to the PivotTable's source table.
+            const dataTable = context.workbook.tables.getItem("DataTable");
+            dataTable.rows.add(null, [["G Farms", "Mango", "Organic", 200, 1000]]);
+            await context.sync();
+          });
+        }
+
+        async function refreshPivotTable() {
```

Comment:
It seems like the point of this snippet is to showcase the refresh() method. Should the refreshPivotPivotTable function be listed first, before addTableRow? 

Also, our recent approach has been to include a sort of thesis statement in the primary/main function in a snippet (the method that is extracted into ref docs), such as: 

"This method shows how to refresh the "Farm Sales" PivotTable to see if new data has been added to the source."